SpatialGDK/Source/SpatialGDK/Public/LoadBalancing/LayeredLBStrategy.h

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "LoadBalancing/AbstractLBStrategy.h"
+
+#include "CoreMinimal.h"
+#include "Math/Box2D.h"
+#include "Math/Vector2D.h"
+
+#include "LayeredLBStrategy.generated.h"
+
+class SpatialVirtualWorkerTranslator;
+class UAbstractLockingPolicy;
+
+DECLARE_LOG_CATEGORY_EXTERN(LogLayeredLBStrategy, Log, All)
+
+USTRUCT()
+struct FLBLayerInfo
+{
+	GENERATED_BODY()
+
+	UPROPERTY()
+	FName Name;
+
+	UPROPERTY(EditAnywhere, Category = "Load Balancing")
+	TSubclassOf<UAbstractLBStrategy> LoadBalanceStrategy;
+
+	UPROPERTY(EditAnywhere, Category = "Load Balancing")
+	TSubclassOf<UAbstractLockingPolicy> LockingPolicy;
+
+
+	FLBLayerInfo() : Name(NAME_None)
+	{
+	}
+};
+
+/**
+ * A load balancing strategy that ...
+ */
+UCLASS(Blueprintable)
```

Comment:
Does this need to be blueprintable?

Reply:
Great catch! We don't need it to be blueprintable and don't want it to be either.